test/unit/Adapter/Driver/Mysqli/ConnectionTest.php

```diff
@@ -133,8 +133,6 @@ public function testConnectionFails()
     protected function createMockMysqli($flags)
     {
         $mysqli = $this->getMockBuilder('\mysqli')->getMock();
-        $mysqli->expects($this->once())
-            ->method('init');
```

Comment:
`init` does not seem to be deprecated: we should probably keep this?

Reply:
it is.
https://wiki.php.net/rfc/deprecations_php_8_1#mysqliinit

Reply:
Indeed, buried under the ginormous RFC, sorry 🤣